ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServersMapperInterface.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.openapi.service.mapper;
+
+public interface ServersMapperInterface {
```

Comment:
Usually we don't name an interface with `Interface` suffix. In this case what we could do is call the interface as `ServerMapper` and then mark the implementation as `ServerMapperImpl`.